extensions/cornerstone/src/init.js

```diff
@@ -102,6 +102,32 @@ export default function init({ servicesManager, configuration }) {
     csTools.BrushTool,
   ];
 
+  /* Measurement Service Events */
+  cornerstone.events.addEventListener(
+    cornerstone.EVENTS.ELEMENT_ENABLED,
+    event => {
+      MeasurementService.subscribe(
+        /* Should subscription event names be handled outside like here? Where to keep available subscription event names? */
+        csTools.EVENTS.MEASUREMENT_ADDED,
+        updatedAnnotation =>
+          console.log(
+            '[subscribe] Measurement added or updated',
+            updatedAnnotation
+          )
+      );
+      const element = event.detail.element;
+      element.addEventListener(csTools.EVENTS.MEASUREMENT_ADDED, event => {
+        console.log('[addOrUpdate] Adding new measurement...', event.detail);
+        MeasurementService.addOrUpdate({
+          id: 1,
```

Comment:
If cornerstone-tools doesn't want to create the Id, we would leave this empty?

Reply:
I think we need the toolstate specific id so we can update this measurement, maybe we cannot even generate ids because the measurement state could be the same?